src/GLib/signals.jl

```diff
@@ -370,16 +370,34 @@ function g_timeout_add(interval::Integer, cb::Function, user_data::CT) where CT
         0, UInt32(interval), callback, ref, deref)
 end
 
+_g_timeout_add_callback(cb::Function) = Cint(cb())
+function g_timeout_add(interval::Integer, cb::Function)
```

Comment:
You might consider making `cb` the first argument so that callers can use `do` syntax.

Reply:
good point, thanks!